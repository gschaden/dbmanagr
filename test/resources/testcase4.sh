db-navigator.py -s skiline
