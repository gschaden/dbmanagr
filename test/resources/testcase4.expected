Id	Title	Subtitle	Autocomplete	
postgres@qa.skiline.net/	postgres@qa.skiline.net/	Connection	postgres@qa.skiline.net/
postgres@qa.skiline.net/skiline	postgres@qa.skiline.net/skiline/	Connection	postgres@qa.skiline.net/skiline/
skiline@bmwqa.skiline.net/	skiline@bmwqa.skiline.net/	Connection	skiline@bmwqa.skiline.net/
skiline@qa.skiline.net/	skiline@qa.skiline.net/	Connection	skiline@qa.skiline.net/
postgres@qa2.skiline.net/	postgres@qa2.skiline.net/	Connection	postgres@qa2.skiline.net/
skiline@qa.skiline.net/skiline	skiline@qa.skiline.net/skiline/	Connection	skiline@qa.skiline.net/skiline/
postgres@bmwqa.skiline.net/skiline	postgres@bmwqa.skiline.net/skiline/	Connection	postgres@bmwqa.skiline.net/skiline/
skiline@qa2.skiline.net/skiline	skiline@qa2.skiline.net/skiline/	Connection	skiline@qa2.skiline.net/skiline/
skiline@livecopy.skiline.net/	skiline@livecopy.skiline.net/	Connection	skiline@livecopy.skiline.net/
skiline@livecopy.skiline.net/skiline	skiline@livecopy.skiline.net/skiline/	Connection	skiline@livecopy.skiline.net/skiline/
postgres@bmwqa.skiline.net/	postgres@bmwqa.skiline.net/	Connection	postgres@bmwqa.skiline.net/
skiline@bmwqa.skiline.net/skiline	skiline@bmwqa.skiline.net/skiline/	Connection	skiline@bmwqa.skiline.net/skiline/
postgres@livecopy.skiline.net/skiline	postgres@livecopy.skiline.net/skiline/	Connection	postgres@livecopy.skiline.net/skiline/

